### ISSUE TITLE
Another file error

### ISSUE THREAD

**Issue by Grimm8094** [gorilla-devs/GDLauncher#1037]
**Describe the bug**
Once more, another issue making it difficult to install any modpack with GD Launcher

**To Reproduce**
Unknown, simply attempt to install a minecraft modpack

**Expected behavior**
install should proceed as normal

**Screenshots**
![Screenshot 2021-08-20 231308](https://user-images.githubusercontent.com/89103555/130310209-2977b3a1-297a-495e-a6e2-91207d6969b9.png)

**Operating System:**
 - Windows10

**Comment by YoYoK3nny** [comment]
There could be an issue with your install.
Did GDLauncher update correctly?
What version are you using?
Have you tried uninstalling and reinstalling? (Don't let the uninstaller reset settings)

The error is referencing a file that doesn't exist. (It may of existed in a previous version)

I'm on v1.1.14 and the only errors I'm getting are related to communication with the mojang servers. Pretty sure that's todo with my internet connection though.

**Comment by Grimm8094** [comment]
> There could be an issue with your install.
> Did GDLauncher update correctly?
> What version are you using?
> Have you tried uninstalling and reinstalling? (Don't let the uninstaller reset settings)
> 
> The error is referencing a file that doesn't exist. (It may of existed in a previous version)
> 
> I'm on v1.1.14 and the only errors I'm getting are related to communication with the mojang servers. Pretty sure that's todo with my internet connection though.

Yeah, it seems to have installed properly and is up-to-date, but the update and installation doesnt look like it's related to my issue.

**Comment by YoYoK3nny** [comment]
Try backing up your user data for GDLauncher, located here by default:
`%USERPROFILE%\AppData\Roaming\gdlauncher_next`
Once backed up, empty the folder. See if it still happens.

Also, does the contents of the following folder look like this:
`%USERPROFILE%\AppData\Local\Programs\gdlauncher\resources`

![image](https://user-images.githubusercontent.com/1491033/130706895-d2419e92-f769-4588-a6d9-f5edb0144cca.png)

**Comment by Grimm8094** [comment]
> Try backing up your user data for GDLauncher, located here by default:
> `%USERPROFILE%\AppData\Roaming\gdlauncher_next`
> Once backed up, empty the folder. See if it still happens.
> 
> Also, does the contents of the following folder look like this:
> `%USERPROFILE%\AppData\Local\Programs\gdlauncher\resources`
> 
> ![image](https://user-images.githubusercontent.com/1491033/130706895-d2419e92-f769-4588-a6d9-f5edb0144cca.png)

Yes, my recourses folder looks just like that, but for backing up the folder, do you mean to back up the gdlauncher_next folder itself and then emptying that whole folder, or something inside?

![Screenshot 2021-09-03 162524](https://user-images.githubusercontent.com/89103555/132066905-0b52473f-c4cc-4b5d-a677-e01f3c07ed6a.png)

**Comment by YoYoK3nny** [comment]
Yes, backup the folder itself and then empty that folder.

When restoring the backup, it is enough to only restore the following folders into `%USERPROFILE%\AppData\Roaming\gdlauncher_next`:
- datastore
- instances
- java

Though with a partial restore, you will need to copy your `Java Custom Arguments` if they are different from the default. (Before emptying the folder)
